common/changes/@bentley/schema-rpcinterface-impl/rpcs-migration_2021-03-30-15-50.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/schema-rpcinterface-impl",
+      "comment": "migrate rpc interface packages from imodeljs-internal",
```

Comment:
Can you update these changelogs to not reference the `imodeljs-internal` repo? Instead say something about it being an initial release.